lib/fs.ts

```diff
@@ -387,6 +387,30 @@ class FSPath {
 	unwatch() {
 		fs.unwatchFile(this.path);
 	}
+
+	async isFile(): Promise<boolean> {
+		return new Promise((resolve, reject) => {
```

Comment:
```suggestion
	async isFile() {
		return new Promise<boolean>((resolve, reject) => {
```

Reply:
I _figured_ this wouldn't work but it was worth a try.

Reply:
I'm not sure what you mean - doing that works for me. But is there a point in doing it?

Reply:
Oh, I mean the two-line suggested change didn't work; you'd have to commit it manually.

As for the point, the main point is that it's DRYer, it uses `Promise` once instead of twice.

Reply:
Ah. Sure.